playgrounds/astro/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "astro-playground",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "clean": "rimraf .turbo && rimraf node_modules && rimraf dist"
+  },
+  "dependencies": {
+    "@astrojs/react": "^1.2.2",
+    "@itwin/itwinui-react": "*",
+    "astro": "~1.6.3",
```

Comment:
Any specific reason for `~`? Would `^` work?

Reply:
i want to keep it `~` because minor version bumps can often have soft breaking changes which might make our build fail